workers/unity/Packages/com.improbable.gdk.mobile/iOS/Utility/iOSDeviceInfo.cs

```diff
@@ -0,0 +1,12 @@
+using UnityEngine;
+
+namespace Improbable.Gdk.Mobile.Ios
+{
+    public static class iOSDeviceInfo
+    {
+        public static bool IsiOSSimulator()
```

Comment:
`isSimulator` is probably fine? has the class is already called `iOSDeviceInfo` and it makes the method easier to read